index.html

```diff
@@ -2376,6 +2376,11 @@ <h3 id="resource">resource</h3>
 
   </section>
 
+  <section
+  data-include="sections/did-method-registry.html"
```

Comment:
this allows us to generate the entire did method registry section as a separate html file